### ISSUE TITLE
Encryption - VPN ?

### ISSUE THREAD

**Issue by merculiv** [YiannisBourkelis/Andama-Remote-Desktop#23]
Would using a VPN client  before running this program be a way of keeping a secure connection?

**Comment by YiannisBourkelis** [comment]
@merculiv Yes I think so, if both computers, local and remote, are using a secure VPN tunel the connection would be secure. The "develop" branch is already using TLS encryption and can be considered secure in theory, however it is still in a very early version, and should be treated as insecure at the moment. There are no binaries yet, so if you would like to try it you should compile the develop branch.

**Comment by merculiv** [comment]
Can you push me the latest one with TLS in a zip?

It will be my forum username at gmx.com

Thanks Yiannis

**Comment by YiannisBourkelis** [comment]
@merculiv just switch to the "develop" branch and download the source code. Just to correct something I wrote before: the encryption is not TLS but AES 256 CBC.
